documentation/index.md

```diff
@@ -1,28 +1,31 @@
-<!--
-SPDX-FileCopyrightText: 2023 Andreas Kahler <mail@andreaskahler.com>
+# Nimble Assembly Guide
 
-SPDX-License-Identifier: CERN-OHL-S-2.0
--->
 
-# Nimble Rack Assembly
+## What is Nimble
 
-The Nimble rack is built from 3d printed parts. There are some standard components, as well as custom trays for all the networking devices that you are using.
+Nimble is an open source, rapidly deployable, wireless mesh network.
 
-This is a [list of needed parts]{BOM}
 
-As a first step you need to 3d print the needed parts
+![](../assets/img/2-23.jpg)
 
-[.](3dprintingparts.md){step}
+Anyone connected to the nimble can video and voice chat, stream videos, share files, build and run eLearning courses and website, create collaborative spreadsheets and documents, read e-books, play games, and so much more… completely offline.
 
-Assembly starts with the frame:
+If a single internet connection is available, it can be plugged into the nimble to provide free or paid internet access to anyone on the network. The offline services and content are always available to users, regardless of internet availability.
 
-[.](frame.md){step}
+Anyone in the world can build their own nimble with locally sourced hardware and 3D-printed components.
 
-You then add the trays and the individual components to the frame:
+You can find out more about Nimble on [our website](https://wakoma.co/nimble/).
 
-[.](components.md){step}
+## Customisation
 
-The assembly is finished by doing the cabling and adding the battery
+Nimble, as an open source project, is designed so that you can tailor it to your own needs and to the hardware that you have available.
 
-[.](finalize.md){step}
+![](../assets/img/Model_M/1.jpg)
 
+We are [working on building a an interface for customising your nimble](https://github.com/Wakoma/nimble/pull/23). In the mean time this is an example of the assembly instructions for a specific configuration.
```

Comment:
```suggestion
We are [working on building an interface for customising your nimble](https://github.com/Wakoma/nimble/pull/23). In the mean time this is an example of the assembly instructions for a specific configuration.
```

Reply:
Also, is "customising" vs "customizing" a UK English vs US English thing?

Reply:
Yeah. We use -ise and -ising rather than -ize and -izing